internal/services/background.service.go

```diff
@@ -110,3 +97,17 @@ func getUnsplashBackgroundImage() models.FrontendBackgroundImageResponse {
 	}
 	return frontendResponse
 }
+
+func GetRandomBackground(source string) models.FrontendBackgroundImageResponse {
+	if source == "unsplash.com" {
+		return getUnsplashBackgroundImage()
+	}
+	return getPexelsBackgroundImage()
+}
+
+func GetBackgroundData() models.FrontendBackgroundImageResponse {
+	// TO-DO
+	// Check in the DB how long ago the image was updated and return the new one or retrieve the old one from the DB
```

Comment:
Better write it in such way: "TODO: Check in...."

Reply:
Fixed: Renamed this comment